private/numerics/functions/elliptic.rkt

```diff
@@ -13,15 +13,10 @@
  (contract-out
```

Comment:
I just remembered: we have `provide/api` to use instead of `contract-out`. I'd prefer we stuck to that.

Start files with:
```
(require mechanics)
(provide/api
  fn
  #:contract (-> Any Any)
  #:doc "A string"

  fn2
  ...
)
```